src/controllers/v1/connections/ConnectionController.ts

```diff
@@ -114,18 +114,28 @@ export class ConnectionController extends Controller {
   }
 
   /**
-   * Deletes a connection record from the connection repository.
-   *
+   * Hangs up an active connection
+   * Optional boolean value to also delete the connection record (default = false)
+   * i.e. /connectionId?delete=true
    * @param connectionId Connection identifier
    */
   @Delete('/:connectionId')
   @Response<NotFoundError['message']>(404)
   @Response<HttpResponse>(500)
-  public async deleteConnection(@Request() req: express.Request, @Path('connectionId') connectionId: UUID) {
+  public async closeConnection(@Request() req: express.Request, @Path('connectionId') connectionId: UUID) {
+    const deleteConnectionRecord: boolean = req.query.delete === 'true'
```

Comment:
I suspect doing it like this instead of using `@Query` means the `delete` option doesn't show in swagger? I might be wrong. @Query would also it could validate it's a boolean